adapters/rest-adapter-core/src/main/java/uk/gov/justice/services/adapter/rest/RestProcessor.java

```diff
@@ -1,28 +1,26 @@
 package uk.gov.justice.services.adapter.rest;
 
-import static java.lang.String.format;
-import static javax.json.JsonValue.NULL;
-import static javax.ws.rs.core.Response.Status.ACCEPTED;
-import static javax.ws.rs.core.Response.Status.INTERNAL_SERVER_ERROR;
-import static javax.ws.rs.core.Response.Status.NOT_FOUND;
-import static javax.ws.rs.core.Response.Status.OK;
-import static javax.ws.rs.core.Response.status;
-import static org.slf4j.LoggerFactory.getLogger;
-import static uk.gov.justice.services.adapter.rest.HeaderConstants.ID;
-
+import org.slf4j.Logger;
 import uk.gov.justice.services.adapter.rest.envelope.RestEnvelopeBuilderFactory;
 import uk.gov.justice.services.messaging.JsonEnvelope;
 
-import java.util.Map;
-import java.util.function.Consumer;
-import java.util.function.Function;
-
 import javax.enterprise.inject.Alternative;
 import javax.json.JsonObject;
 import javax.ws.rs.core.HttpHeaders;
 import javax.ws.rs.core.Response;
+import java.util.Map;
+import java.util.function.Consumer;
+import java.util.function.Function;
 
-import org.slf4j.Logger;
+import static java.lang.String.format;
+import static javax.json.JsonValue.NULL;
+import static javax.ws.rs.core.Response.Status.*;
+import static javax.ws.rs.core.Response.status;
+import static org.slf4j.LoggerFactory.getLogger;
+import static uk.gov.justice.services.common.http.HeaderConstants.ID;
+import static uk.gov.justice.services.messaging.logging.HttpMessageLoggerHelper.toHttpHeaderTrace;
+import static uk.gov.justice.services.messaging.logging.JsonEnvelopeLoggerHelper.toEnvelopeTraceString;
+import static uk.gov.justice.services.messaging.logging.LoggerUtils.trace;
```

Comment:
Please use the coding style config for the project to avoid these imports getting reordered. Plus we don't use wildcard imports.